RiotSharp/Http/Interfaces/IRequestCreator.cs

```diff
@@ -0,0 +1,11 @@
+﻿using System.Collections.Generic;
+using System.Net.Http;
+using RiotSharp.Misc;
+
+namespace RiotSharp.Http.Interfaces
+{
+    public interface IRequestCreator
+    {
+        HttpRequestMessage CreateGetRequest(Region region, string apiEndpoint, List<string> addedArguments, bool useHttps = true);
```

Comment:
Here might be needed functions for `PUT` and `POST` for the tournament endpoint.

Reply:
You're right. I hadn't written them yet since this only covered the standard requester which is only used by the status api. I'll include it in the branch you suggested.